tests/common/helpers.py

```diff
@@ -117,7 +117,7 @@ async def _started_cb(**kwargs):
         controller.stopped.set()
 
 
-async def create_shared_workspace(name, creator, *shared_with):
+async def create_shared_workspace(name, creator: UserFS, *shared_with: UserFS) -> EntryID:
```

Comment:
creator & shared_with should be typed `UserFS | LoggedCore` (see line 132 & 145)

Reply:
done :+1: